lib/msal-common/apiReview/msal-common.api.md

```diff
@@ -2215,6 +2215,17 @@ const emptyInputScopesError = "empty_input_scopes_error";
 // @public (undocumented)
 const emptyInputScopeSet = "empty_input_scopeset";
 
+// Warning: (ae-missing-release-tag) "EncodingTypes" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+// Warning: (ae-missing-release-tag) "EncodingTypes" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
nit: Duplicate line.

Reply:
So this is actually expected since there's two exports for EncodingTypes. There's one warning for each export.